test/test_autograd.py

```diff
@@ -3073,6 +3074,9 @@ def test_svd_no_singularvectors(self):
             torch.autograd.backward([u, s, v], [torch.ones_like(u), torch.ones_like(s), torch.ones_like(v)])
 
     def test_no_grad_copy(self):
+        if torch.autograd.get_num_threads_per_device() > 1:
```

Comment:
nit: to be consistent with other skipping mechanism, you might want to do something similar to other skipping mechanisms. For example what we do for multi gpu.
```
@unittest.skipIf(not RUN_CUDA_MULTI_GPU, "multi-GPU not supported")
def test_foo(self):
```